### PR TITLE
Better way to handle process.env variables

### PR DESCRIPTION
**Pull request by toddw**

### CODE REVIEW

File: src/config/getWebpackClientConfig.js [hunk 1]
```diff
@@ -74,6 +74,23 @@ export default function (appRoot, appConfigFilePath, isProduction) {
...
+              "babel-plugin-gluestick"
+            ],
+            presets: [
+              "react",
+              "es2015",
+              "stage-0"
+            ]
+          },
+          include: [
+            path.join(process.cwd(), "src/config/application.js"),
```
Comment (christinebrass) [inline src/config/getWebpackClientConfig.js]: Can we use `appRoot` for `process.cwd()` here?

Comment (toddw) [reply src/config/getWebpackClientConfig.js]: yeah totally! I didn't even know what that was

Comment (toddw) [reply src/config/getWebpackClientConfig.js]: updated

**Comment by christinebrass** [comment]
👍

**Pull request merged**